src/components/schedule-page/edit-page/edit-page-toolbar.component.tsx

```diff
@@ -84,6 +84,7 @@ export function EditPageToolbar({ closeEdit }: EditPageToolbarOptions): JSX.Elem
 
   function askForSavingChanges(): void {
     if (anyChanges()) setIsSaveModalOpen(true);
+    else closeEdit(false);
```

Comment:
Wydaje mi się, że wystarczy po prostu wywołać tę funkcję i nie potrzeba dodawać `false`.
Bo to leci jakoś tak
![image](https://user-images.githubusercontent.com/21079319/111835001-ae252600-88f4-11eb-988a-b93a3a83a9ec.png)